core/app/c/(public)/[communitySlug]/public/Header.tsx

```diff
@@ -1,8 +1,8 @@
 import type { ReactNode } from "react";
 
 import { Button } from "ui/button";
-import { Bookmark } from "ui/icon";
 
+import { FormLinkCopyButton } from "~/app/components/forms/FormLinkCopyButton";
```

Comment:
I think you'll need to dynamically import this or else you'll get "window is not defined" errors on the server:

```
NEXT_RUNTIME is `edge` and NODE_ENV is `development`; skipping OTEL + Sentry registration.
 GET /c/croccroc/public/forms/review/fill 500 in 469ms
 ⨯ core/app/components/forms/FormLinkCopyButton.tsx (9:28) @ FormLinkCopyButton
 ⨯ ReferenceError: window is not defined
    at FormLinkCopyButton (./core/app/components/forms/FormLinkCopyButton.tsx:9:28)
digest: "2216156825"
   7 |
   8 | export const FormLinkCopyButton = (props: Props) => {
>  9 | 	return <CopyButton value={window.location.href} {...props} />;
     | 	                          ^
  10 | };
  11 |
 GET /favicon.ico?favicon.412d60c0.ico 200 in 86ms
```

I had to do the same thing for the [FormCopyButton](https://github.com/pubpub/platform/blob/em/form-copy-link/core/app/c/%5BcommunitySlug%5D/forms/%5BformSlug%5D/edit/page.tsx#L19-L22)